ionic_user_interface/src/views/Help.vue

```diff
@@ -38,111 +33,196 @@
         <ion-button
           target="_blank"
           href="mailto:routemakergithub@gmail.com"
+          title="Email us!"
           color="light"
           size="medium"
           class="logo-button"
         >
           <ion-icon :icon="mail" size="medium"></ion-icon>
           Feedback
         </ion-button>
-
         <br />
         <br />
         <br />
         <strong>-- How To Use --</strong>
-
-        <p class="lead my-4">1. Upload the photo of a climbing wall</p>
-        <ion-img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_1.jpg`" />
-        <br />
-        <p class="lead my-4">2. The holds will be marked automatically</p>
-        <ion-img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_2.jpg`" />
-        <br />
-        <p class="lead my-4">3. Create a custom route by following the prompts</p>
-        <ion-img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_3.jpg`" />
-        <br />
-        <p class="lead my-4">4. Post to share with other gym goers</p>
-        <ion-img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_4.jpg`" />
-        <br />
-        <p class="lead my-4">5. Vote, grade and comment on other's routes</p>
-        <ion-img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_5.jpg`" />
-        <br />
+        <div class="slides-container">
+          <div class="slides-arrow left-align" @click="prevSlide">
+            <ion-icon :icon="chevronBack"></ion-icon>
+          </div>
+          <div class="slides-arrow right-align" @click="nextSlide">
+            <ion-icon :icon="chevronForward"></ion-icon>
+          </div>
+          <ion-slides pager="true" ref="ionSlides">
+            <ion-slide>
+              <div>
+                <p>1. Upload the photo of a climbing wall</p>
```

Comment:
These are different font sizes from other `p` tags

Reply:
Not sure what you mean, but if you want to change the font-size u can just change